moveit_core/utils/CMakeLists.txt

```diff
@@ -28,5 +33,10 @@ ament_target_dependencies(moveit_test_utils
   srdfdom
   urdfdom
   urdfdom_headers
+<<<<<<< HEAD
+=======
+  rclcpp
+  fmt
+>>>>>>> 3df6b2813 (Depend on `rsl::rsl` as a non-Ament dependency (#2578))
```

Comment:
```suggestion
```
I think nothing has to change here either. `main` uses `rclcpp` and `fmt` but this branch apparently does not so we don't need to add them.